diffpy/pdfmorph/pdfmorphapp.py

```diff
@@ -253,31 +258,39 @@ def main():
         if opts.baselineslope is None:
             refpars.append("baselineslope")
             config["baselineslope"] = -0.5
+
+    def nn_value(val, name):
```

Comment:
is there a reason you are defining this inside main?  if not, pull it out for greater readability

Reply:
will do